chainerx_cc/chainerx/CMakeLists.txt

```diff
@@ -129,6 +143,12 @@ if(${CHAINERX_BUILD_TEST})
         stack_vector_test.cc
         strides_test.cc
         thread_local_state_test.cc
+        testing/array_check.cc
+        testing/array_test.cc
+        testing/gtest_main.cc
+        testing/routines.cc
+        testing/routines_test.cc
+        testing/util.cc
```

Comment:
I'm wondering if these additions are really necessary. 🤔

Reply:
These changes were actually committed and pushed by mistake (I needed them in my local environment for the tests to compile) so I can just revert them if that would be better.

Reply:
On second thought this seems related to the `--whole-archive` shenanigan.

Reply:
Yes indeed it was...